\chapter{Goal of collecting genomic data from cancers}

\section{Context}
In the start of the second millennium, scientific inquiry as a whole
has remained relatively absent from political discourse and relatedly,
the collective consciousness of the nation. Part of this may be due to
the end of the Cold War around 20 years prior, which removed the
impetus for international scientific competition; another part may be
due to the sufficient progress science has made in many fields,
leading the way to technologies and engineering to take the forefront
and to translate discoveries into general comfort.

Health remains an important area, although even here science seems to
have sufficiently addressed most aspects – infectious diseases and
most non-infectious diseases are now much less deadly than they were
in the recent past. To those with sufficient means in the United
States and most western countries, developments based on scientific
inquiry serves them well.

Arguably the biggest exception to this is cancer; in the last 100
years, cancer mortality as a whole has only very slightly
decreased. One reason for this may be the mechanistic uniqueness of
the disease in every individual.

Encouragingly, the sequencing of the human genome in 2001 and the
subsequent decreasing cost of genomic sequencing methods has yielded
some promise in cancer treatment possibilities. In particular,
machines that can gather sequencing information from tumors (“genomic
data”) may be able to be used for an increasing amount of the
population, financially, and these might be able to lead to the
appropriate handling of uniqueness to decrease mortality.

In 2015, United States’ President Barack Obama announced the Precision
Medicine initiative during his State of the Union address. During the
same address in 2016, he announced the National Cancer Moonshot
Initiative. Both were rightfully meant to focus attention on this
problem.

        In this thesis, I make use of
        three large-scale multi-institutional consortium-based efforts and one
        smaller-scale intra-institutional effort to idnetify impactful genomic
        alternations using novel tools and approaches. First, I
        outline the overall goals of the collection of this data,
        which are twofold: 


\section{Investigation}
	In order to successfully use genomics data on a personal level
        for cancer treatment, much needs to be collected in order to
        increase our understanding of how that genomics data is
        related to properties of tumors and treatment decisions. There
        are several ongoing and recently completed large-scale efforts
        to collect this data in order to achieve this. These can be
        seen as investigations towards understanding specific areas of
        unknown about the mechanisms of cancer onset and progression.


        \subsection{Identification of Molecular Subtypes}

	One area of unknown is the existence of molecular subtypes of
        tumors – groups of tumors based on the molecular aberrations
        they share. This is in contrast with the classical tissue
        definition of tumors, which is based on the type of tissue
        from which they originated. The Cancer Genome Atlas (TCGA)\cite{mclendon_comprehensive_2008}, a
        major consortium which gathered 11,000 tumor samples from 33
        classically-defined cancer types, aims to help assess this
        difference. I later describe identification of impactful
        features from one tumor type contained in TCGA. 
        
        Analyses under the TCGA, which gather sequencing data and compare it
        across tumor types, have made some findings that begin to suggest that
        a significant fraction tumors that are defined classically would be
        better assessed based on their molecular similarity with a
        non-classically equivalent tumor.

        One area of success in molecular subtype definition has been
        gliomas. Gliomas are one of the most common forms of brain
        cancer, comprising 30\% of brain and central nervous system
        and 80\% of all malignant brain tumors. Gliomas are
        classically classified by World Health Organization (WHO)
        grade (II through IV for adults), and the glial subtype that
        the tumor cells most resemble based on examining tumor tissue
        from biopsies under the microscope (histopathology):
        oligodendroglioma, astrocytoma, or oligoastrocytoma for tumors
        appearing similar to oligodendrocytes, astrocytes, or both,
        respectively. 
        

        Somewhat expectedly based on prior related research which suggested
        yet was not able to provide sufficient evidence for a definitive
        conclusion, TCGA researchers studying sequencing information from over
        300 lower-grade glioma (LGG) tumors identified three molecular
        subtypes of the tumors based on sequencing information. LGG tumors
        were defined based on histopathologists’ assessments of tumor tissue
        as being indicative of grade II of any histopathology.


        This led to finding consistent subgroups of patterns across
        all types of  sequencing information. Importantly, these
        subgroups were found to correlate more closely with important
        patient outcomes than the classical classifications (grade and
        glial subtype resembled). Interestingly, researchers were also
        able find that subtypes, while evidenced by much sequencing
        information across the entire genome, were well defined by a
        very small number nuclear changes as well, suggesting possible
        targets for therapies. Researchers also identified that one
        molecular subtype appeared very similar to tumors whose
        classical classification would have indicated a higher grade,
        and these tumors had similarly poor prognosis as these
        higher-grade tumors.


        

        In addition to finding subtypes within one specific classical tumor
        diagnosis category and comparing those subtypes with other classical
        categories in an ad-hoc manner, TCGA has also implemented the search
        for subtypes with so-called Pan-cancer analysis\cite{weinstein_cancer_2013} studies. Based on
        clustering, researchers assessed classes of tumors in a classical
        category-agnostic fashion. This is consistent with the molecular
        understandings of (initiation and onset of tumor tissue
        (oncogenesis). In particular, the same general mutational patterns
        involving the same classes of genes are likely to have the ability to
        initiate tumor-like properties across classical tumor
        types.

        In the context of cancer genomics, mutations are defined as changes in the DNA of a
        tumor that are not also present in the DNA of non-tumor
        cells. These are sometimes termed ``somatic mutations,'' to
        underscore their occurrence within the tumor tissue of the
        individual in which they are detected, and are contrasted with
        ``germline mutations.


        The major finding here in terms of subtypes was the stratification of
        classical tumor categories on a spectrum from tumors with a high
        mutational burden (many small aberrations in genomic DNA present in
        tumor tissue as compared to normal tissue derived from blood) to those
        with a high copy number change burden (many large-scale aberrations in
        genomic DNA). Interestingly, no classical category appeared to have
        high amounts of both type of mutation; this may point to similarities
        in oncogenesis mechanisms in tissues that are similar in terms of
        mutational burden type, and two general oncogenic tumor
        categories.
        
        \subsubsection{Assessing Clinical Utility of Molecular Subtypes}

        A further area of investigation is whether subtypes identified
        based on consistent molecular patterns have clinical utility.

        
        One promising result is in gliomas, where the finding that molecular subtypes were more
        accurate at distinguishing clinical outcomes can now be used by
        clinicians to treat patients with the privilege of having had
        their genome sequenced; the WHO is in the process of updating
        the standard of care for LGG patients, which will formalize
        this new ability to treat patients more effectively. In
        particular, those with a prognosis similar to higher grade
      tumors based on molecular subtyping can be treated
        accordingly; this may involve more aggressive use of
        chemotherapy and/or radiotherapy.

        Towards this end, the Cancer Cell Line
        Encyclopaedia (CCLE)\cite{barretina_cancer_2012} and the Cancer
        Genome Project (CGP) \cite{ledford_end_2015} are testing a variety
        of new targeted therapies \textit{in vitro} in a relatively knowledge-agnostic
        approach. I use data from both consortiums towards this goal,
        as well. Targeted therapies are pharmaceutical agents that
        interrupt the activity of a particular molecular process,
        typically by interfering with a protein. As part of the
        rational development of cancer therapies, they are typically
        chosen in a strategic way to interrupt key pathways that are
        important in particular types. The molecular similarities
        between tumors of different types justifies the relatively
        unbiased approaches used by both studies, which contrasts with
        a one-tissue-type, one-drug approach.
        
        In particular, for  example, drugs targeting the epidermal growth factor receptor
        (EGF) pathway have been shown to be effective against multiple
        different tumor tissues of origin
        \cite{barretina_cancer_2012}.


        \subsection{Assessing Relative Impact of Genetic and
          Epigeneitc Changes}

        A second area of open investigation is the role of epigenetics
        in oncogenesis, tumor progression, and
        metastasis. Epigenetic changes are consist of non-base-pair
        DNA changes to the genome based on modification of the
        genome's surroundings; in particular, chromatin can be opened
        are closed via a myraid of mechanisms, such as histone
       modifications and DNA methylation \todo{cite}.

        These have an unknown degree of impact;
        it is not known, for example, whether such changes
        (``epimutations'') are sufficient for oncogenesis \todo{cite}. 

        Towards that end, Dr. Joseph Costello, Ph.D. is spearheading a
        University of California, San Francisco-based approach to
        gather both genetic and epigenetic data on LGG tumors. He has
        found already DNA methylation-based epigenetic changes
        correspond closely with alternations in DNA base
        pairs\cite{johnson_mutational_2014}. 

        \subsection{Identification of Driver Mutations}

        Another general goal of collecting genomic data from tumors,
        and one of the first goals, has been the identification of
        ``driver'' mutations. Driver mutations are DNA changes that
        are  ``causally implicated
        in oncogenesis''\cite{stratton_cancer_2009}. Driver mutations
        are typically validated via \textit{in vitro} and \textit{in vivo}
        experiments. The number of possible driver mutations is as
        endless as the number of possible mutations, so therefore
        candidate driver mutations need to be established in order to
        narrow down the search space of possible drivers. 

        One common prioritzation technique is to identify somatic
        mutations that occur more often than what might be expected
        based on chance (``recurrent mutations''). This technique
        requires statistical power, necessitating, in turn, the
        collection of many tumor samples. Several methods have been
        adapted, such as \cite{leiserson_pan-cancer_2015}. Another 

        


        